source/module_io/mulliken_charge.cpp

```diff
@@ -7,27 +7,21 @@
 
      12/Oct/2018  Released by Feng Qi
      03/2023/     Refactored by Yuyang Ji
+     18/04/2023   Convert to namespace by Yu Liu
 
 ***********************************************************************/
 
 #include "mulliken_charge.h"
 #include "module_hamilt_pw/hamilt_pwdft/global.h"
 #include "module_base/global_function.h"
 #include "module_base/global_variable.h"
-#include "module_base/matrix.h"
-#include "module_base/complexmatrix.h"
 #include "module_base/scalapack_connector.h"
 #include "write_orb_info.h"
-#ifdef __LCAO
-#include "module_hamilt_lcao/hamilt_lcaodft/local_orbital_charge.h"
-#include "module_hamilt_lcao/hamilt_lcaodft/LCAO_gen_fixedH.h"
-#include "module_hamilt_lcao/hamilt_lcaodft/LCAO_matrix.h"
-#include "module_hamilt_lcao/hamilt_lcaodft/LCAO_hamilt.h"
-#include "module_hamilt_lcao/hamilt_lcaodft/global_fp.h"
-#endif
-#include "module_cell/module_neighbor/sltk_atom_arrange.h"
 
-ModuleBase::matrix Mulliken_Charge::cal_mulliken(const std::vector<ModuleBase::matrix> &dm,
+namespace Mulliken_Charge
```

Comment:
Can we use ModuleIO to be consistent with other output files?

Reply:
OK, I have done it.